opendevin/runtime/client/runtime.py

```diff
@@ -145,7 +146,7 @@ async def _init_container(
                 )
                 volumes = None
 
-            logger.info(f'run_as_devin: `{self.config.run_as_devin}`')
+            logger.info(f'_init_container run_as_devin: {self.config.run_as_devin}')
```

Comment:
How about we log the entire config to help with general purpose debugging?

Reply:
The full config seems like overkill to me, tbh. And as long as we don't see the output in CI anyways, it's not helping us.

Reply:
I think only logging the `self.config.run_as_devin` seems only helpful for debugging the issue we discussed earlier (e.g., run_as_devin related), but not for other issues.

If we are make it `logger.debug`, we might as well as log more rather than less to save people the hassle to turn them on again?

Reply:
Fair enough, will change that with next commit.

Reply:
Is this changed?